config/locales/en.yml

```diff
@@ -1,33 +1,33 @@
-# Files in the config/locales directory are used for internationalization
-# and are automatically loaded by Rails. If you want to use locales other
-# than English, add the necessary files in this directory.
-#
-# To use the locales, use `I18n.t`:
-#
-#     I18n.t 'hello'
-#
-# In views, this is aliased to just `t`:
-#
-#     <%= t('hello') %>
-#
-# To use a different locale, set it with `I18n.locale`:
-#
-#     I18n.locale = :es
-#
-# This would use the information in config/locales/es.yml.
-#
-# The following keys must be escaped otherwise they will not be retrieved by
-# the default I18n backend:
-#
-# true, false, on, off, yes, no
-#
-# Instead, surround them with single quotes.
-#
-# en:
-#   'true': 'foo'
-#
-# To learn more, please read the Rails Internationalization guide
-# available at https://guides.rubyonrails.org/i18n.html.
-
+---
 en:
-  hello: "Hello world"
+  action-login: Login
+  action-register: Register
+  bride: Bride
+  celebrant: Celebrant
+  create_project_button: Start composing the booklet!
+  get-started-link: Get started!
+  groom: Groom
+  hello: Hello world
+  page-title: Our Marriage Booklet
+  pages-new_project-email_address-description: Your email address will never be shared without your permission. Wedding booklet projects you create will be associated with your email address.
+  pages-new_project-email_address-label: My email address is
```

Comment:
We should take advantage of YAML. Rather than 

```yaml
pages-new_project-email_address-label: My email address is
```
we should prefer something like
```yaml
pages:
  new_project:
    email_address_label: My email address is
    marriage_date_label: The wedding will take place on (date and time)
```

Which can be referenced in code like:
```ruby
I18n.t('pages.new_project.email_address_label')
```

Reply:
done in https://github.com/JohnRDOrazio/marriage-booklet/commit/15adb8e2a68257c50f8371de2a41ed5faea42bde